src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -14139,22 +14188,6 @@ public void Parameter_Attribute_Insert()
                 capabilities: EditAndContinueTestHelpers.Net6RuntimeCapabilities);
         }
 
-        [Fact]
-        public void Parameter_Attribute_Insert_SupportedByRuntime_NonCustomAttribute()
```

Comment:
Why delete this one? I don't think there is another test that covers the deny list of attributes

Reply:
Mistake. Reverted.